pkg/resource/notebook_instance/custom_sdk_api.go

```diff
@@ -0,0 +1,15 @@
+package notebook_instance
+
+import (
+	svcsdk "github.com/aws/aws-sdk-go/service/sagemaker"
+)
+
+func (rm *resourceManager) stopNotebookInstance(r *resource) error {
```

Comment:
1. would it be possible to indicate in the filename what it does - I did not find a file with this name in any other resource - is this a convention you saw in other services ?

2. Could add a more detailed comment on top of this method as to why this is needed ?

Reply:
Also, could you update the description to mention this file.

Reply:
This file is just for functions that call the api. Included a comment.